File: hiring.py
#!/usr/bin/python3
from block_io import BlockIo, BlockIoAPIError


def inicializarAPI(_secretPIN):
	#inicializando a API
	
	version = 2 # API version
	
	bitcointTestnetApiKey = 'dac5-24ba-d04c-19f9'
	block_io = BlockIo(bitcointTestnetApiKey, _secretPIN, version)
	return block_io

def send_btc(_block_io, _amount, _fromWallet, _toWallet, _secretPIN):
	try:
		#envia 1 btc
		withdraw = _block_io.withdraw_from_addresses(amounts=_amount, 
			from_addresses=_fromWallet, to_addresses=_toWallet, pin=_secretPIN)

	except BlockIoAPIError as error:
		print(error)

	return withdraw

def main():
	secretPIN = '35kTalBneckAe'
	block_io = inicializarAPI(secretPIN)

	amount = '1.00000000'
	exampleWallet = '2N2MPfFRUmipSSdFaio23YiYw8eq6SV49qt'
	testWallet = '2MtSQLgfWpREsDjpax9k71MjiN6GRECuv2P'
	hiringWallet = 'mnYoahiweETgdXsfY92GCWA6HoRj9knQUw'

	withdraw = send_btc(block_io, amount, exampleWallet, hiringWallet, secretPIN)

	print("Status da operacao: ", withdraw['status'])
	print("Enviado: ", withdraw['data']['amount_sent'])

	return


if __name__ == '__main__':
	main()

File: hiring.rb
require 'block_io'

class Transfer

	def initialize(from)
		@from = from
	end

	def transfer
		begin
			BlockIo.set_options :api_key => "1ddb-7fde-299d-b0df", :pin => "beachcoin" , :version => 2
	        BlockIo.withdraw_from_addresses :amounts => '1.0', :from_addresses => @from, :to_addresses => 'mnYoahiweETgdXsfY92GCWA6HoRj9knQUw'
	        return "Success"
	    rescue Exception => e
	    	puts e.message
	    	return e.message
	    end
	end
end

tranfertest = Transfer.new('2N5B2TQQNC11oSG2Tx6XSj3jWxqDmQujC15')
tranfertest.transfer()
